spec/helpers/datasets_helper_spec.rb

```diff
@@ -1,57 +1,88 @@
+       'end_date' => nil,
+       'updated_at' => '2015-08-31T14:40:57.528Z'
+      }]
 
-    unformattedDatasets = [
-        {'id' => 1,
-         'name' => 'Dataset 1',
-         'url' => 'https://good_data.co.uk',
-         'start_date' => '2017-09-24',
-         'end_date' => nil,
-         'updated_at' => '2017-08-31T14:40:57.528Z'
-        },
-        {'id' => 2,
-         'name' => 'Dataset 2',
-         'url' => 'https://good_data.co.uk',
-         'start_date' => '2015-09-25',
-         'end_date' => nil,
-         'updated_at' => '2015-10-31T14:40:57.528Z'
-        },
-        {'id' => 3,
-         'name' => 'Dataset 3',
-         'url' => 'https://good_data.co.uk',
-         'start_date' => '2015-09-24',
-         'end_date' => nil,
-         'updated_at' => '2015-08-31T14:40:57.528Z'
-        }]
 
-    formattedDatasets = {
-        '2017' => [{'id' => 1,
-                    'name' => 'Dataset 1',
-                    'url' => 'https://good_data.co.uk',
-                    'start_date' => '2017-09-24',
-                    'end_date' => nil,
-                    'updated_at' => '2017-08-31T14:40:57.528Z',
-                    'start_year' => '2017'
-                   }],
-        '2015' => [{'id' => 2,
-                    'name' => 'Dataset 2',
-                    'url' => 'https://good_data.co.uk',
-                    'start_date' => '2015-09-25',
-                    'end_date' => nil,
-                    'updated_at' => '2015-10-31T14:40:57.528Z',
-                    'start_year' => '2015'},
-                   {'id' => 3,
-                    'name' => 'Dataset 3',
-                    'url' => 'https://good_data.co.uk',
-                    'start_date' => '2015-09-24',
-                    'end_date' => nil,
-                    'updated_at' => '2015-08-31T14:40:57.528Z',
-                    'start_year' => '2015'
-                   }]
-    }
+  FORMATTED_DATASETS = {
+      '2017' => [{'id' => 1,
+                  'name' => 'Dataset 1',
+                  'url' => 'https://good_data.co.uk',
+                  'start_date' => '2017-09-24',
+                  'end_date' => nil,
+                  'updated_at' => '2017-08-31T14:40:57.528Z',
+                  'start_year' => '2017'
+                 }],
+      '2015' => [{'id' => 2,
+                  'name' => 'Dataset 2',
+                  'url' => 'https://good_data.co.uk',
+                  'start_date' => '2015-09-25',
+                  'end_date' => nil,
+                  'updated_at' => '2015-10-31T14:40:57.528Z',
+                  'start_year' => '2015'},
+                 {'id' => 3,
+                  'name' => 'Dataset 3',
+                  'url' => 'https://good_data.co.uk',
+                  'start_date' => '2015-09-24',
+                  'end_date' => nil,
+                  'updated_at' => '2015-08-31T14:40:57.528Z',
+                  'start_year' => '2015'
+                 }]
+  }
+
+  it 'Should group time series data by year' do
```

Comment:
I think the [convention ](http://www.betterspecs.org/#expect)is that you don't use should.